sigs.yaml

```diff
@@ -2363,21 +2363,23 @@ sigs:
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/OWNERS
 workinggroups:
-- dir: wg-apply
-  name: Apply
+- dir: wg-api-expression
+  name: API Expression
   mission_statement: >
-    Improve the state of declarative object management by fixing `kubectl apply`,
-    moving the logic from the client to the server. This move also enables new/fixed
-    features included in the scope of this working-group: diff, dry-run and prune.
-
-    Resources can be found in [this Google drive folder](https://drive.google.com/drive/folders/1wlpgkS2gFZXdp4x2WlRsfUBxkFlt2Gx0)
+    Improve API expressiveness and support API evolution.
 
   stakeholder_sigs:
   - API Machinery
-  - CLI
-  label: apply
+  - Architecture
+  label: api-expression
   leadership:
     chairs:
+    - github: apelisse
+      name: Antoine Pelisse
+      company: Google
+    - github: kwiesmueller
+      name: Kevin Wiesmueller
+      company: //SEIBERT/MEDIA GmbH
```

Comment:
Won't be in my capacity as an employee (mostly) so i'm not sure if that belongs here.

Reply:
Whether to list it or not is up to you & your employer :)